### ISSUE TITLE
Suggest creating "Description:" heading in Event details modal.

### ISSUE THREAD

**Issue by intelagense** [Together-100Devs/Together#307]
## Please provide a summary of the feature
![image](https://user-images.githubusercontent.com/18331781/215569643-d386bad0-4ef6-4497-af52-6b7b600b7c65.png)

The event details modal inserts the word "Description: " as additional text before the actual description from the event creator. Semantically it would be better to include this in a heading tag, preferably above the text box area. 

This would also improve the user experience when recreating events. Currently, when copying and pasting a previous event description in order to recreate a new event description the "Description:" can be copied and inadvertently duplicated as well.

Example:
>"Description: Description: An event where people gather to do things."

## Are there any potential issues you foresee with this feature
Aesthetics in regards to #17 👀

## Which Branch should this fix be Pulled into?
PR into [branchName]

*If you're interested in working on this feature, please comment so you can be assigned*

**Comment by intelagense** [comment]
link to some relevant code: 
https://github.com/Caleb-Cohen/Together/blob/343d7378306ad752cbada653b4de1f6d6dab829b/client/src/features/modal/EventModal.jsx#L64-L66

**Comment by Caleb-Cohen** [comment]
Just a heads up this has been solved in a future redesign in #285. Redesign will probably take some time so if someone wanted to take this on it will be available until #285 is pulled into development.

**Comment by vguzman812** [comment]
Hi. This seems like an easy fix. I would like to be assigned this if possible. I already have a solved branch ready that will close this issue.

**Comment by cblanken** [comment]
This issue should be closed. It was resolved by PR #444

**Comment by emmebravo** [comment]
@DevinCLane this tix needs to be closed
